src/python/pants/notes/2.13.x.md

```diff
@@ -0,0 +1,73 @@
+# 2.12.x Release Series
+
+## 2.13.0.dev0 (May 05, 2022)
+
+### New Features
+
+* Add lz4 support to archive extraction ([#15245](https://github.com/pantsbuild/pants/pull/15245))
+
+* Implement `tailor` goal for Helm charts ([#15191](https://github.com/pantsbuild/pants/pull/15191))
+
+* Write virtualenvs for Python tools as part of `export` ([#15098](https://github.com/pantsbuild/pants/pull/15098))
+
+### User API Changes
+
+* Deprecate `[golang].expected_version` in favor of more flexible `[golang].minimum_expected_version` ([#15306](https://github.com/pantsbuild/pants/pull/15306))
+
+* Upgrade Pex to 2.1.84. ([#15288](https://github.com/pantsbuild/pants/pull/15288))
+
+* Upgrade Pex to 2.1.83. ([#15281](https://github.com/pantsbuild/pants/pull/15281))
+
+* Upgrade Pex to 2.1.82. ([#15200](https://github.com/pantsbuild/pants/pull/15200))
+
+* Upgrade Pex to 2.1.80. ([#15179](https://github.com/pantsbuild/pants/pull/15179))
```

Comment:
This should be manually removed, as only the latest release actually matters for users.

Reply:
done.